client/store/variations/resolvers.js

```diff
@@ -0,0 +1,27 @@
+/** @format */
+/**
+ * External dependencies
+ */
+// import apiFetch from '@wordpress/api-fetch';
+import { dispatch } from '@wordpress/data';
+import { stringify } from 'qs';
+import { isEmpty } from 'lodash';
+
+export default {
+	async getVariations( state, query ) {
+		try {
+			const params = isEmpty( query ) ? '' : '?' + stringify( query );
+			// @TODO: Use /reports/variations when it becomes available
+			// const variations = await apiFetch( {
+			// 	path: '/wc/v3/reports/variations' + params,
+			// } );
+			const variations = await fetch(
+				'https://virtserver.swaggerhub.com/peterfabian/wc-v3-api/1.0.0/reports/variations' + params
```

Comment:
Not sure how I feel about using the swagger apis

Reply:
How come?

Reply:
They give us a very static version of the data.  When we merge UI that rely on the swagger apis it is non-functional.  Additionally there is no pressure to get those API's working because the UI "working"

Reply:
I'm okay with approving, just wanted to see what others thought.  I have three PR's ready to roll that I could use this method for.

Reply:
Per our discussion in the hangout yesterday, this is fine for now, we just need to call it out in release notes as not yet complete.